components/gitpod-db/src/typeorm/migration/1685447855319-RevertOrgOnlyMigrationForBuiltinUser.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class RevertOrgOnlyMigrationForBuiltinUser1685447855319 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        // Unset all user values set by UserToTeamMigrationService
+        queryRunner.query(
+            "UPDATE d_b_user SET usageAttributionId = '', additionalData = JSON_REMOVE(additionalData, '$.isMigratedToTeamOnlyAttribution', '$.shouldSeeMigrationMessage') WHERE id IN ('builtin-user-workspace-probe-0000000', 'builtin-user-agent-smith-0000000', 'f071bb8e-b5d1-46cf-a436-da03ae63bcd2')",
```

Comment:
Interesting that these are not actually valid UUIDs, but that's secondary to this PR